### PR TITLE
Update for new Mobile SDKs: Part 1 of 4

### PR DESCRIPTION
**Pull request by mauricesharp-okta**
Update documentaiton to use new versions of mobile SDKs. This includes updating links, prose, and code.

Part 1 has small changes to a number of articles including links, minor prose, and moving some articles to the Classic archive category. See the notes for the first commit for a detailed list of changes including paths to the articles.

### CODE REVIEW

File: packages/@okta/vuepress-site/code/android/index.md [hunk 1]
```diff
@@ -26,30 +26,26 @@ These resources walk you through adding user authentication to your Android app
 	</li>
 </ul>
 
-## Integrate with Okta using the Okta-hosted Sign-In Widget
+## Add sign-in with Okta to your app
 
-These SDKs help you integrate with Okta by redirecting to the Okta Sign-In Widget using OpenID Connect (OIDC) client libraries.
+There are two ways to integrate sign-in with Okta to your app. The easiest to integrate is the redirect model which uses the Okta-hosted Sign-In Widget. You open the widget, Okta processes the sign-in attempt, and sends the result to your app. The other is the embedded model, where your manages each step in the sign-in flow. See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/ios/main/) for more detail.
```
Comment (JayNewstrom) [inline packages/@okta/vuepress-site/code/android/index.md:31]:
```suggestion
There are two ways to integrate sign-in with Okta to your app. The easiest to integrate is the redirect model which uses the Okta-hosted Sign-In Widget. You open the widget, Okta processes the sign-in attempt, and sends the result to your app. The other is the embedded model, where you manage each step in the sign-in flow. See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/ios/main/) for more detail.
```

File: packages/@okta/vuepress-site/code/ios/index.md [hunk 2]
```diff
@@ -25,32 +25,26 @@ These resources walk you through adding user authentication to your iOS app in m
 	</li>
 </ul>
 
-## Integrate with Okta using the Okta-hosted Sign-In Widget
+## Add sign-in with Okta to your app
 
-These SDKs help you integrate with Okta by redirecting to the Okta Sign-In Widget using OpenID Connect (OIDC) client libraries.
+There are two ways to integrate sign-in with Okta to your app. The easiest to integrate is the redirect model which uses the Okta-hosted Sign-In Widget. You open the widget, Okta processes the sign-in attempt, and sends the result to your app. The other is the embedded model, where your manages each step in the sign-in flow. See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/ios/main/) for more detail.
```
Comment (JayNewstrom) [inline packages/@okta/vuepress-site/code/ios/index.md:30]:
```suggestion
There are two ways to integrate sign-in with Okta to your app. The easiest to integrate is the redirect model which uses the Okta-hosted Sign-In Widget. You open the widget, Okta processes the sign-in attempt, and sends the result to your app. The other is the embedded model, where you manage each step in the sign-in flow. See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/ios/main/) for more detail.
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 3]
```diff
@@ -116,7 +116,10 @@ You should now have a better understanding of how a sign-in session is shared be
...
 To learn more about our Mobile OpenID Connect (OIDC) SDKs and sample apps:
 
-* [Okta OIDC for iOS](https://github.com/okta/okta-oidc-ios)
-* [Okta OIDC for Android](https://github.com/okta/okta-oidc-android)
-* [Okta iOS Browser Sign-in Sample App](https://github.com/okta/samples-ios/tree/master/browser-sign-in)
-* [Kotlin Sign-In Sample App for Android](https://github.com/okta/samples-android/tree/master/sign-in-kotlin)
\ No newline at end of file
+Android:
+- [Okta Mobile SDK for Kotlin](https://github.com/okta/okta-mobile-kotlin).
+-  [Okta Classic Android Sign-In Kotlin Sample App](https://github.com/okta/samples-android/tree/legacy-samples/sign-in-kotlin).
```
Comment (JayNewstrom) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:121]:
```suggestion
- [Okta Classic Android Sign-In Kotlin Sample App](https://github.com/okta/samples-android/tree/legacy-samples/sign-in-kotlin).
```

**Review by JayNewstrom — APPROVED** [review]

File: packages/@okta/vuepress-site/code/android/index.md (hunk 1 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/android/index.md:31]: Done, in next commit.

File: packages/@okta/vuepress-site/code/ios/index.md (hunk 2 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/ios/index.md:30]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md (hunk 3 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:121]: Done, in next commit.

File: packages/@okta/vuepress-site/code/android/index.md [hunk 4]
```diff
@@ -26,30 +26,26 @@ These resources walk you through adding user authentication to your Android app
 	</li>
 </ul>
 
-## Integrate with Okta using the Okta-hosted Sign-In Widget
+## Add sign-in with Okta to your app
 
-These SDKs help you integrate with Okta by redirecting to the Okta Sign-In Widget using OpenID Connect (OIDC) client libraries.
+There are two ways to integrate sign-in with Okta to your app. The easiest to integrate is the redirect model which uses the Okta-hosted Sign-In Widget. You open the widget, Okta processes the sign-in attempt, and sends the result to your app. The other is the embedded model, where you manage each step in the sign-in flow. See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/ios/main/) for more detail.
```
Comment (mikenachbaur-okta) [inline packages/@okta/vuepress-site/code/android/index.md:31]: This is the `android` markdown file, but the link to the IDX docs references the iOS version.

**Review by mikenachbaur-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/code/android/index.md (hunk 4 above)
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/android/index.md:31]:
```suggestion
There are two ways to integrate sign-in with Okta to your app. The easiest to integrate is the redirect model that uses the Okta-hosted Sign-In Widget. You open the widget, Okta processes the sign-in attempt, and then sends the result to your app. The other is the embedded model, where you manage each step in the sign-in flow. See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/ios/main/).
```

File: packages/@okta/vuepress-site/code/android/index.md [hunk 5]
```diff
@@ -26,30 +26,26 @@ These resources walk you through adding user authentication to your Android app
...
 
-[Android redirect authentication sample app](https://github.com/okta/samples-android): See [Browser Sign In](https://github.com/okta/samples-android/tree/master/browser-sign-in) for a redirect configuration.
+### Use the redirect model
 
-## Integrate with Okta using embedded Sign-In Widget and SDKs
+This method works with both Okta Identity Engine and Okta Classic.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/android/index.md:35]:
```suggestion
This method works with both Okta Identity Engine and Okta Classic Engine.
```

File: packages/@okta/vuepress-site/code/android/index.md [hunk 6]
```diff
@@ -26,30 +26,26 @@ These resources walk you through adding user authentication to your Android app
...
 
-* [Identity Engine Java SDK](https://github.com/okta/okta-idx-java)
-* [Android embedded authentication with SDK sample app](https://github.com/okta/okta-idx-android)
 
-Okta Classic:
+### Use the embedded model
 
-* [Okta OIDC Android SDK](https://github.com/okta/okta-oidc-android#requirements)
-* [Android SDK Source](https://github.com/okta/okta-oidc-android)
+This method works only with Okta Identity Engine.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/android/index.md:44]:
```suggestion
This method works only with Identity Engine.
```

File: packages/@okta/vuepress-site/code/ios/index.md [hunk 7]
```diff
@@ -25,32 +25,26 @@ These resources walk you through adding user authentication to your iOS app in m
 	</li>
 </ul>
 
-## Integrate with Okta using the Okta-hosted Sign-In Widget
+## Add sign-in with Okta to your app
 
-These SDKs help you integrate with Okta by redirecting to the Okta Sign-In Widget using OpenID Connect (OIDC) client libraries.
+There are two ways to integrate sign-in with Okta to your app. The easiest to integrate is the redirect model which uses the Okta-hosted Sign-In Widget. You open the widget, Okta processes the sign-in attempt, and sends the result to your app. The other is the embedded model, where you manage each step in the sign-in flow. See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/ios/main/) for more detail.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/ios/index.md:30]:
```suggestion
There are two ways to integrate sign-in with Okta to your app. The easiest to integrate is the redirect model that uses the Okta-hosted Sign-In Widget. You open the widget, Okta processes the sign-in attempt, and then sends the result to your app. The other is the embedded model, where you manage each step in the sign-in flow. See [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/ios/main/).
```

File: packages/@okta/vuepress-site/code/ios/index.md [hunk 8]
```diff
@@ -25,32 +25,26 @@ These resources walk you through adding user authentication to your iOS app in m
...
 
-[iOS redirect authentication sample app](https://github.com/okta/samples-ios): See [Browser sign in](https://github.com/okta/samples-ios/tree/master/browser-sign-in) for a redirect configuration.
+### Use the redirect model
 
-## Integrate with Okta using embedded Sign-In Widget and SDKs
+This method works with both Okta Identity Engine and Okta Classic.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/ios/index.md:34]:
```suggestion
This method works with both Okta Identity Engine and Okta Classic Engine.
```

File: packages/@okta/vuepress-site/code/ios/index.md [hunk 9]
```diff
@@ -25,32 +25,26 @@ These resources walk you through adding user authentication to your iOS app in m
...
 
-* [Identity Engine Swift SDK](https://github.com/okta/okta-idx-swift)
-* [iOS embedded authentication with SDK sample app](https://github.com/okta/okta-idx-swift/tree/master/Samples/EmbeddedAuthWithSDKs)
 
-Okta Classic OIDC library for iOS (CocoaPod):
+### Use the embedded model
 
-* [OktaOidc on CocoaPods](https://cocoapods.org/pods/OktaOidc)
-* [Okta iOS OIDC SDK Source](https://github.com/okta/okta-oidc-ios)
+This method works only with Okta Identity Engine.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/code/ios/index.md:43]:
```suggestion
This method works only with Identity Engine.
```

File: packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/main/android/samplecode.md [hunk 10]
```diff
@@ -1 +1 @@
-See our [Android custom UI example](https://github.com/okta/samples-android/tree/master/sign-in-kotlin).
\ No newline at end of file
+See the Classic sample [Android custom UI example](https://github.com/okta/samples-android/tree/legacy-samples/sign-in-kotlin) in the legacy branch.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/main/android/samplecode.md:1]:
```suggestion
See the Classic Engine sample [Android custom UI example](https://github.com/okta/samples-android/tree/legacy-samples/sign-in-kotlin) in the legacy branch.
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/android/samplecode.md [hunk 11]
```diff
@@ -1,3 +1,3 @@
-See our [Android Samples repo](https://github.com/okta/samples-android) for multiple samples.
+See the Classic samples in the legacy branch of the [Android Samples repository](https://github.com/okta/samples-android/tree/legacy-samples) for multiple samples.
 
-You configure the [Android Sign-In Kotlin Sample App](https://github.com/okta/samples-android/tree/master/sign-in-kotlin) in the [Set up the first mobile app](#set-up-the-first-mobile-app) section.
\ No newline at end of file
+You configure the [Okta Classic Android Sign-In Kotlin Sample App](https://github.com/okta/samples-android/tree/legacy-samples/sign-in-kotlin) in the [Set up the first mobile app](#set-up-the-first-mobile-app) section.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/android/samplecode.md:3]:
```suggestion
You configure the [Okta Classic Engine Android Sign-In Kotlin Sample App](https://github.com/okta/samples-android/tree/legacy-samples/sign-in-kotlin) in the [Set up the first mobile app](#set-up-the-first-mobile-app) section.
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 12]
```diff
@@ -116,7 +116,10 @@ You should now have a better understanding of how a sign-in session is shared be
...
 To learn more about our Mobile OpenID Connect (OIDC) SDKs and sample apps:
 
-* [Okta OIDC for iOS](https://github.com/okta/okta-oidc-ios)
-* [Okta OIDC for Android](https://github.com/okta/okta-oidc-android)
-* [Okta iOS Browser Sign-in Sample App](https://github.com/okta/samples-ios/tree/master/browser-sign-in)
-* [Kotlin Sign-In Sample App for Android](https://github.com/okta/samples-android/tree/master/sign-in-kotlin)
\ No newline at end of file
+Android:
+- [Okta Mobile SDK for Kotlin](https://github.com/okta/okta-mobile-kotlin).
+- [Okta Classic Android Sign-In Kotlin Sample App](https://github.com/okta/samples-android/tree/legacy-samples/sign-in-kotlin).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:121]:
```suggestion
- [Okta Classic Engine Android Sign-In Kotlin Sample App](https://github.com/okta/samples-android/tree/legacy-samples/sign-in-kotlin).
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/ios/samplecode.md [hunk 13]
```diff
@@ -1,3 +1,3 @@
-See our [iOS Samples repo](https://github.com/okta/samples-ios) for multiple samples.
+See the Okta Classic samples in the legacy branch of the [iOS Samples repository](https://github.com/okta/samples-ios/tree/legacy-samples) for multiple samples.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/ios/samplecode.md:1]:
```suggestion
See the Classic Engine samples in the legacy branch of the [iOS Samples repository](https://github.com/okta/samples-ios/tree/legacy-samples) for multiple samples.
```

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/ios/samplecode.md [hunk 14]
```diff
@@ -1,3 +1,3 @@
...
 
-You configure the [iOS Browser Sign-In Sample App](https://github.com/okta/samples-ios/tree/master/browser-sign-in) in the [Set up the first mobile app](#set-up-the-first-mobile-app) section. You'll need to install [CocoaPods](https://cocoapods.org/) to install the dependencies for the iOS Browser Sign-In sample app, using `sudo gem install cocoapods`.
\ No newline at end of file
+You configure the [Okta Classic iOS Browser Sign-In Sample App](https://github.com/okta/samples-ios/tree/legacy-samples/browser-sign-in) in the [Set up the first mobile app](#set-up-the-first-mobile-app) section.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/ios/samplecode.md:3]:
```suggestion
You configure the [Okta Classic Engine iOS Browser Sign-In Sample App](https://github.com/okta/samples-ios/tree/legacy-samples/browser-sign-in) in the [Set up the first mobile app](#set-up-the-first-mobile-app) section.
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/main/android/samplecode.md [hunk 15]
```diff
@@ -1 +1 @@
-[Android samples repo](https://github.com/okta/samples-android)
\ No newline at end of file
+Classic samples in the legacy branch of the [Android Samples repository](https://github.com/okta/samples-android/tree/legacy-samples)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/main/android/samplecode.md:1]:
```suggestion
Classic Engine samples in the legacy branch of the [Android Samples repository](https://github.com/okta/samples-android/tree/legacy-samples)
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/main/ios/samplecode.md [hunk 16]
```diff
@@ -1 +1 @@
-[iOS samples repo](https://github.com/okta/samples-ios)
\ No newline at end of file
+Classic samples in the legacy branch of the [iOS Samples repository](https://github.com/okta/samples-ios/tree/legacy-samples)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/main/ios/samplecode.md:1]:
```suggestion
Classic Engine samples in the legacy branch of the [iOS Samples repository](https://github.com/okta/samples-ios/tree/legacy-samples)
```

File: packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/android/samplecode.md [hunk 17]
```diff
@@ -1 +1 @@
-We have a sample available in our [Android samples repo](https://github.com/okta/samples-android/tree/master/browser-sign-in).
\ No newline at end of file
+See the [Browser sign-in sample](https://github.com/okta/samples-android/tree/legacy-samples/browser-sign-in) for Classic.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/android/samplecode.md:1]:
```suggestion
See the [Browser sign-in sample](https://github.com/okta/samples-android/tree/legacy-samples/browser-sign-in) for Classic Engine.
```

File: packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/ios/samplecode.md [hunk 18]
```diff
@@ -1 +1 @@
-We have a sample available in our [iOS samples repo](https://github.com/okta/samples-ios/tree/master/browser-sign-in-and-biometric-storage).
\ No newline at end of file
+See the [Browser sign in and biometric storage sample](https://github.com/okta/samples-ios/tree/legacy-samples/browser-sign-in-and-biometric-storage) for Classic.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/ios/samplecode.md:1]:
```suggestion
See the [Browser sign in and biometric storage sample](https://github.com/okta/samples-ios/tree/legacy-samples/browser-sign-in-and-biometric-storage) for Classic Engine.
```

**Review by susanharper-okta — COMMENTED** [review]
couple comments

File: packages/@okta/vuepress-site/code/android/index.md (hunk 4 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/android/index.md:31]: Done, in next commit.

File: packages/@okta/vuepress-site/code/android/index.md (hunk 5 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/android/index.md:35]: Done, in next commit.

File: packages/@okta/vuepress-site/code/android/index.md (hunk 6 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/android/index.md:44]: Done, in next commit.

File: packages/@okta/vuepress-site/code/ios/index.md (hunk 7 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/ios/index.md:30]: Done, in next commit.

File: packages/@okta/vuepress-site/code/ios/index.md (hunk 8 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/ios/index.md:34]: Done, in next commit.

File: packages/@okta/vuepress-site/code/ios/index.md (hunk 9 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/ios/index.md:43]: Done, in next commit.

File: packages/@okta/vuepress-site/code/ios/index.md [hunk 19]
```diff
@@ -60,13 +54,14 @@ Okta-hosted Sign-In Widget guide:
...
+- [Overview of the mobile Identity Engine SDK](/docs/guides/mobile-idx-sdk-overview/ios/main/)
+- [Get set up with Identity Engine sample apps and embedded use cases](/docs/guides/oie-embedded-common-org-setup/ios/main/)
 
 Other guides:
 
-* [Implementing the Authorization Code flow with PKCE](/docs/guides/implement-grant-type/authcodepkce/main/)
-* [Add an identity provider (includes social login)](/docs/guides/identity-providers/)
-* [Validate access tokens](/docs/guides/validate-access-tokens)
-* [Validate ID tokens](/docs/guides/validate-id-tokens)
+- [Implementing the Authorization Code flow with PKCE](/docs/guides/implement-grant-type/authcodepkce/main/)
```
[the remark this answers is not in the document]
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/ios/index.md:62]: Done, in next commit.

[the remark this answers is not in the document]
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/ios/index.md:62]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/main/android/samplecode.md (hunk 10 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/main/android/samplecode.md:1]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/main/ios/samplecode.md [hunk 20]
```diff
@@ -1 +1 @@
-See our [iOS custom UI example](https://github.com/okta/samples-ios/tree/master/custom-sign-in/).
\ No newline at end of file
+See the Classic sample [iOS custom UI example](https://github.com/okta/samples-ios/tree/legacy-samples/custom-sign-in/) in the legacy branch of the archive.
```
[the remark this answers is not in the document]
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/main/ios/samplecode.md:1]: Done, and fixed other occurrences of this too.

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/android/samplecode.md [hunk 21]
```diff
@@ -1,3 +1,3 @@
-See our [Android Samples repo](https://github.com/okta/samples-android) for multiple samples.
+See the Classic samples in the legacy branch of the [Android Samples repository](https://github.com/okta/samples-android/tree/legacy-samples) for multiple samples.
```
[the remark this answers is not in the document]
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/android/samplecode.md:1]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/android/samplecode.md (hunk 11 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/android/samplecode.md:3]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md [hunk 22]
```diff
@@ -42,7 +42,7 @@ Single Sign-On (SSO) allows users to authenticate once and access multiple resou
 * **Session SSO**: Session SSO cookies are written for the authenticated user, which eliminates further prompts when the user switches applications during a particular session. However, if a particular session ends, the user is prompted for their credentials again.
 * **Persistent SSO**: Persistent SSO cookies are written for the authenticated user, which eliminates further prompts when the user switches applications for as long as the persistent SSO cookie is valid.
 
-The difference between persistent SSO and session SSO is that persistent SSO can be maintained across different sessions. Persistent SSO is disabled by default in Okta. To share a sign-in session with native mobile apps, you need to enable persistent SSO. 
+The difference between persistent SSO and session SSO is that persistent SSO can be maintained across different sessions. Persistent SSO is disabled by default in Okta. To share a sign-in session with native mobile apps, you need to enable persistent SSO.
```
[the remark this answers is not in the document]
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:45]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md (hunk 12 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/index.md:121]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/ios/samplecode.md (hunk 13 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/ios/samplecode.md:1]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/ios/samplecode.md (hunk 14 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/shared-sso-android-ios/main/ios/samplecode.md:3]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/main/ios/samplecode.md (hunk 16 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-users-out/main/ios/samplecode.md:1]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/android/samplecode.md (hunk 17 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/android/samplecode.md:1]: Done, in next commit.

File: packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/ios/samplecode.md (hunk 18 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/main/ios/samplecode.md:1]: Done, in next commit.

File: packages/@okta/vuepress-site/code/android/index.md (hunk 4 above)
Comment (mauricesharp-okta) [reply packages/@okta/vuepress-site/code/android/index.md:31]: Done, fixed in next commit.

**Pull request merged**